src/styles/tailwind.css

```diff
@@ -25,22 +25,3 @@
     display: none;
   }
 }
-
-.prose code {
-  background: transparent;
-  border: none;
-  color: #ffffff;
-  box-shadow: none;
-  font-size: 14px;
-}
-@property --discord-button-angle {
-  syntax: "<angle>";
-  inherits: false;
-  initial-value: 0deg;
-}
-
-@property --discord-button-x {
-  syntax: "<length>";
-  inherits: false;
-  initial-value: 0;
-}
```

Comment:
Can you restore the changes in this file? I believe that it can remain the same as it was before the PR.